ClassesHomework/Cat.cs

```diff
@@ -6,10 +6,59 @@
 
 namespace ClassesHomework
 {
-    // Write your Cat class here
-    //class Cat
-    //{
-        
-    //}
+    class Cat : Animal
+    {
+        public Cat()
+        {
+            IsSitting = false;
+            Age = 0;
+
+        }
+
+        public string Breed { get; private set; }
+        public int Age { get; private set; }
+        public bool IsSitting { get; private set; }
+
+        public Cat(string breed)
+        {
+            Breed = breed;
+        }
+
+        public Cat(int age)
+        {
+            Age = age;
+        }
+
+        public Cat(string breed, int age)
+        {
+            Breed = breed;
+            Age = age;
+        }
+
+        public string Speak()
+        {
+            return "meow!";
+        }
+
+        public string Speak(string sayThisCat)
+        {
+            return sayThisCat;
+        }
+
+        public bool Sit()
```

Comment:
Is Sit() supposed to return some information, or is it supposed to do/change something? Same with Stand()